ocs_ci/ocs/bucket_utils.py

```diff
@@ -1251,3 +1258,168 @@ def get_bucket_available_size(mcg_obj, bucket_name):
     resp = bucket_read_api(mcg_obj, bucket_name)
     bucket_size = resp["storage"]["values"]["free"]
     return bucket_size
+
+
+class BucketPolicyOps:
+    def __init__(self, bucket_factory, mcg_obj, interface="OC"):
+        from ocs_ci.ocs.resources.objectbucket import OBC
+
+        self.obc_name = bucket_factory(amount=1, interface=interface)[0].name
+        self.mcg_obj = mcg_obj
+        self.obc_obj = OBC(self.obc_name)
+
+    def bucket_policy_put(self):
+        logger.info("Bucket policy: Add")
+        from ocs_ci.ocs.resources.bucket_policy import (
```

Comment:
Please move imports to the top of this file

Reply:
Faced cyclic import issue